addons/binding/org.openhab.binding.avmfritz/META-INF/MANIFEST.MF

```diff
@@ -37,5 +39,6 @@ Import-Package:
  org.openhab.binding.avmfritz.handler,
  org.osgi.framework,
  org.osgi.service.component,
- org.slf4j
+ org.slf4j,
+ tec.uom.se
```

Comment:
Is this import necessary? I don't get any hits on the when I search for imports of that package.

Reply:
That package actually must not be imported by bindings.

Reply:
If I do so I get an error while compiling. What has to be done to resolve that? Is there a relation to the `WATT_HOUR` unit used in https://github.com/openhab/openhab2-addons/pull/3477/files#diff-bd72c5c4d5010dd42a9b6940a70ad50bR220? Or did I something wrong? If I remove that unit. It compiles w/o any errors.
```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.0.0:compile (default-compile) on project org.openhab.binding.avmfritz: Compilation failure: Compilation failure:
[ERROR] ~/openhab2-addons/addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/ahamodel/PowerMeterModel.java:
[ERROR] /**
[ERROR] ^
[ERROR] The type tec.uom.se.AbstractSystemOfUnits cannot be resolved. It is indirectly referenced from required .class files
```

Reply:
@htreu That sounds like a bug in ESH, doesn't it?

Reply:
One more thing: It seems to be a problem with the static import of `org.eclipse.smarthome.core.library.unit.SmartHomeUnits.*`.

Reply:
investigating. stay tuned.

Reply:
Thats actually a bug: https://github.com/eclipse/smarthome/issues/5413

Reply:
@cweitkamp the workaround is to **not** statically import `SmartHomeUnits.*` or even single fields like `SmartHomeUnits.WATT_HOUR` but use the simple class import and the qualified static access in the code.

Reply:
@htreu Thank you. I have changed it for now.